frontend/src/pages/NutriScore.jsx

```diff
@@ -6,57 +6,49 @@ function NutriScore() {
-          les protéines, les fibres, ainsi que les pourcentages de la valeur
-          quotidienne (% VQ) de ces éléments. En combinant ces informations, le
-          Nutri-Score évalue la qualité nutritionnelle globale d'un aliment, en
-          prenant en compte les nutriments bénéfiques et les éléments à limiter.
-          Ce système d'étiquetage est devenu un outil précieux pour les
-          consommateurs soucieux de leur santé, les aidant à faire des choix
-          alimentaires plus éclairés dans un environnement où les informations
-          nutritionnelles peuvent parfois être complexes à interpréter.
+          The Nutri-Score takes into account several key nutritional values,
+          such as calories, saturated fats, sugars, salt, proteins, fibers, as
+          well as percentages of the value daily (% DV) of these elements. By
+          combining this information, the Nutri-Score evaluates the overall
+          nutritional quality of a food, in taking into account beneficial
+          nutrients and elements to limit. This labeling system has become a
+          valuable tool for health-conscious consumers, helping them make
+          choices more informed food in an environment where information
+          Nutritional information can sometimes be complex to interpret.
         </p>
         <img className="text-line" src={Line} alt="line" />
       </div>
       <div className="text-container">
-        <h2 className="text-title">
-          Un Outil Clé pour une Alimentation Plus Saine
-        </h2>
+        <h2 className="text-title">A Key Tool for a Healthier Diet</h2>
         <p className="text-description">
-          Le Nutri-Score est un exemple de la manière dont les valeurs
-          nutritionnelles sont mises en pratique pour promouvoir une
-          alimentation plus saine. Il contribue à sensibiliser les consommateurs
-          aux qualités nutritionnelles des produits alimentaires et encourage
-          les fabricants à améliorer la composition nutritionnelle de leurs
-          produits. En fin de compte, les valeurs nutritionnelles et le
-          Nutri-Score sont des outils puissants pour promouvoir la santé
-          publique en aidant les individus à faire des choix alimentaires plus
-          informés, favorisant ainsi une alimentation équilibrée et une
-          meilleure qualité de vie.
+          The Nutri-Score is an example of how values nutritional practices are
+          put into practice to promote healthy healthier diet. It helps raise
+          consumer awareness nutritional qualities of food products and
+          encourages manufacturers to improve the nutritional composition of
+          their products. Ultimately, the nutritional values and Nutri-Score are
+          powerful tools to promote health public by helping individuals make
+          better food choices informed, thus promoting a balanced diet and
+          better quality of life.
```

Comment:
Cadeau => The Nutri-Score is an example of how nutritional values are put into practice to promote a healthier diet. It contributes to raising awareness among consumers about the nutritional qualities of food products and encourages manufacturers to improve the nutritional composition of their products. Ultimately, nutritional values and the Nutri-Score are powerful tools for promoting public health by assisting individuals in making more informed dietary choices, thereby promoting a balanced diet and a better quality of life.